src/express/common/ganache-utils.js

```diff
@@ -10,7 +12,31 @@ const EthAmount = '10'
 // we implemented this workaround waiting for a migration to hardhat
 // (see internal issue https://polygon.atlassian.net/browse/POS-1869)
 export async function fundGanacheAccounts(doc) {
-  const machine0 = doc.devnetBorHosts[0]
+  let machine0
+  if (doc === undefined || doc == null) {
+    dotenv.config({ path: `${process.cwd()}/.env` })
+
+    const devnetType =
+      process.env.TF_VAR_DOCKERIZED === 'yes' ? 'docker' : 'remote'
+
+    doc = await loadDevnetConfig(devnetType)
+  } else {
+    machine0 = doc.devnetBorHosts[0]
+  }
+
+  if (
+    doc.borChainId !== 137 && // mainnet
```

Comment:
Would it make sense to have those `borChainId` values in a const array so that it's easier to maintain / extend?